release-notes/v0.7.0.md

```diff
@@ -0,0 +1,150 @@
+
+![Folding Range](v0.7.0/java.folding.range.gif)
+
+## Debugging
+
+### Display Logical Structure of Collections
+
+The debugger is now showing the logical structure of lists and maps, instead of the physical layout of the collections.
+
+If you prefer the physical layout view, you can go back by setting `java.debug.settings.showLogicalStructure` to `false`.
+
+![Logical Structure of Collections](v0.7.0/java.debug.logical.structure.gif)
+
+### Highlight Exceptions in Editor
+
+Exceptions are now highlighted with extra info in the editor window. Before that, you need to hover on the exception to see details. Now the most important info is presented to you right at where it occurs.
+
+![Highlight Excpetions in Editor](v0.7.0/java.debug.exception.view.gif)
+
+### Go to Definition by Clicking Stack Trace in Debug Console
+
+When there's an exception, you can click on the stack trace to see the definition of the function calls.
+
+![Go to Definition by Clicking Stack Trace in Debug Console](v0.7.0/debug.gtd.stack.trace.gif)
+
+## Maven
+
+Maven extensions released some interesting features and improvements to improve the productivity.
+
+### Debug Maven Goals
+
+To debug Maven goals, just right click on a goal and start debugging. The Maven extension will call the Java debugger with the right parameters. This is handy and super time-saving.
+
+![Debug Maven Goals](v0.7.0/maven.debug.goals.gif)
+
+### New Command to Add a Dependency
+
+Maven extension released a new command `Maven: Add a Dependency` (or `maven.project.addDependency`) to help add a new dependency to `pom.xml`. The process is interactive.
+
+![Maven Add Dependency](v0.7.0/maven.add.dependency.gif)
+
+## Troubleshooting
+
+### JDK Acquisition Guide
+
+If you are experiencing the problem of setting up the JDK, we've got you covered. The JDK Acquisition Guide follows the same logic used by the Java language server to check for a JDK installation on your machine. When the check fails, the guide will automatically show and recommend a JDK distribution that is compatible with your OS and CPU architecture.
+
+![JDK Acquisition Guide](v0.7.0/jdk.acquisition.guide.gif)
+
+## Thank You
+
+Here are more changes and bug fixes. Kudos to all the contributors!
+
+- [@snjeza](https://github.com/snjeza)
+    - [Langauge Server] Download java sources lazily [eclipse/eclipse.jdt.ls#979](https://github.com/eclipse/eclipse.jdt.ls/issues/979)
+    - [Langauge Server] Optimize compilation unit computations [eclipse/eclipse.jdt.ls#980](https://github.com/eclipse/eclipse.jdt.ls/issues/980)
+    - [Langauge Server] Optimize server initialization [eclipse/eclipse.jdt.ls#981](https://github.com/eclipse/eclipse.jdt.ls/issues/981)
+- [@testforstephen](https://github.com/testforstephen)
+    - [Debugger] Debug console does not autocomplete when a .class file is open [Microsoft/vscode-java-debug#535](https://github.com/Microsoft/vscode-java-debug/issues/535)
```

Comment:
Add a bug fix for java-debug:
-	Auto shorten the command line for the case "CreateProcess error=206, The filename or extension is too long". See https://github.com/Microsoft/vscode-java-debug/issues/110